index.html

```diff
@@ -108,7 +107,7 @@ <h3>Multi-Post Stories Gain+Glory</h3>
                                 <li><a href="#">JavaScript</a></li>
                                 <li><a href="#">HTML</a></li>
                             </ul>
-                            <button class="button">See Project</button>
+                            <button class="button" id="btnPopup1">See Project</button>
```

Comment:
- [ ] I think it is one of the main requirements is to create the original cards that you already created in HTML you should  create it by using javascript so kindly create it by that would help you learning how to create HTML by js 🔥🚀💯

![image](https://user-images.githubusercontent.com/58553711/188764823-a7ac4e1d-31a3-4a48-bf57-e03997c86971.png)